build.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+multipleBuildsPath="../builds/"
+individualBuildAllFiles="build/*"
+
+craBatteryConsiderateLoading="cra-battery-considerate-loading"
+craDeviceClassAwareLoading="cra-device-class-aware-loading"
+craMemoryConsiderateLoading="cra-memory-considerate-loading"
+craMemoryConsiderateLoadingSketchfab="cra-memory-considerate-loading-sketchfab"
+craNetworkAwareCodeSplitting="cra-network-aware-code-splitting"
+craNetworkAwareComponent="cra-network-aware-component"
+craNetworkAwareDataFetching="cra-network-aware-data-fetching"
+craUaAwareCodeSplitting="cra-ua-aware-code-splitting"
+reactMovieNetworkAwareComponents="react-movie-network-aware-components"
+reactShrineNetworkAwareCodeSplitting="react-shrine-network-aware-code-splitting"
+
```

Comment:
Nothing to change with respect to this feedback, but leaving here for later:

In a typical review, I would normally say we should figure out how to define the list of subprojects in an array and iterate over the build and deploy steps for them in a loop as the steps are currently quite similar. That said, I actually **like** the flexibility we get with your current setup in case we need really granular control at some point.

Reply:
Yes, I totally agree with you.
I was eager to use loop and array of projects because the current version looks not professional and even hardcoded.
But on the other hand I thought I should not spend much time on secondary problems.
So just for now I described this bash script to the meaningful extend because I thought I could get back to this refinement when I get some free or extra time.
TODO is remained here :)